pkgs/by-name/ku/kubectl-graph/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-graph";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "steveteuber";
+    repo = "kubectl-graph";
+    rev = "v${version}";
+    sha256 = "sha256-5N1eC8J0nHEgFUCHEn5b3kUDj6MWejouQBKkdJKsaAo=";
+  };
+
+  vendorHash = "sha256-fvn+CoOyMRjsIemMRXitMjTlbbhjrlDIHu398b4/ZZ0=";
+
+  meta = with lib; {
+    description = "A kubectl plugin to visualize Kubernetes resources and relationships.";
```

```suggestion
    description = "kubectl plugin to visualize Kubernetes resources and relationships";
```

See https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes